impeller/compiler/compiler_backend.h

```diff
@@ -11,23 +11,28 @@
 #include "flutter/fml/macros.h"
 #include "spirv_glsl.hpp"
 #include "spirv_msl.hpp"
+#include "spirv_sksl.h"
 
 namespace impeller {
 namespace compiler {
 
 struct CompilerBackend {
   using MSLCompiler = std::shared_ptr<spirv_cross::CompilerMSL>;
   using GLSLCompiler = std::shared_ptr<spirv_cross::CompilerGLSL>;
-  using Compiler = std::variant<MSLCompiler, GLSLCompiler>;
+  using SkSLCompiler = std::shared_ptr<CompilerSkSL>;
+  using Compiler = std::variant<MSLCompiler, GLSLCompiler, SkSLCompiler>;
```

Comment:
Ooh, sorry about this. This is a cleanup that is tracked in https://github.com/flutter/flutter/issues/106376.